config/playground/helm/ml/helm-opensearch.yaml

```diff
@@ -110,8 +110,8 @@ opensearchJavaOpts: "-Xmx2G -Xms2G"
 
 resources:
   requests:
-    cpu: "2"
-    memory: "16Gi"
+    cpu: "16"
```

Comment:
Do we need to increase OpenSearch container as well? I thought you guys just need to increase Machine learning Node 's CPU and Memory, right? Thanks

Reply:
No Tao, we need to increase both the data node and ml node, because when ingesting our nq dataset into the data node 32GB is not enough